Tests/Build-Bicep.Tests.ps1

```diff
@@ -1,6 +1,6 @@
 BeforeAll {
     $ScriptDirectory = Split-Path -Path $PSCommandPath -Parent
```

Comment:
I don't think these are necessary now just before Module import same goes for supportfiles, they could use PSScriptRoot, or ?

Reply:
I agree, we should be able to use $PSScriptRoot everywhere